### PR TITLE
Rebase 2.0 with GE GitHub Enterprise mirror

### PR DESCRIPTION
**Pull request by dkrypt**
This PR introduces latest changes from developers contributing in GE GitHub Enterprise Repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +50,7 @@ The deployment requires a pair of client(license) ID and owner's hash which can
 Instance | API | Branch | Description
 -- | --- | --- | ---
 [dc-portal](https://dc-portal.run.aws-usw02-dev.ice.predix.io/v1.2beta/dc) | [swagger](https://dc-portal.run.aws-usw02-dev.ice.predix.io/v1.2beta/assets/swagger-ui/) | [Repo](https://github.com/dc-release/dc-portal/tree/v1beta) | staging
-[dc-portal-1x](https://dc-portal-1x.run.aws-usw02-dev.ice.predix.io/v1.2beta/dcsc) | [swagger](https://dc-portal-1x.run.aws-usw02-dev.ice.predix.io/v1.2beta/assets/swagger-ui/) | [Repo](https://github.com/paskantishubham/dc-portal/tree/v1beta) | dev
```
Comment (ayasuda-ge) [inline README.md:46]: please add back and reconcile.

File: README.md [hunk 2]
```diff
@@ -43,9 +50,7 @@ The deployment requires a pair of client(license) ID and owner's hash which can
...
-[dc-portal-ext](https://dc-portal-ci-ext.digitalconnect.apps.ge.com/v1.2beta/dc) | [swagger](https://dc-portal-ci-ext.digitalconnect.apps.ge.com/v1.2beta/assets/swagger-ui/) | [Repo](https://github.com/dc-release/dc-portal/tree/v1beta) | EKS dev with external ingress
```
Comment (ayasuda-ge) [inline README.md:47]: please add back and reconcile.

File: README.md [hunk 3]
```diff
@@ -43,9 +50,7 @@ The deployment requires a pair of client(license) ID and owner's hash which can
...
-[dc-portal-int](https://dc-portal-ci-int.digitalconnect.apps.ge.com/v1.2beta/dc) | [swagger](https://dc-portal-ci-int.digitalconnect.apps.ge.com/v1.2beta/assets/swagger-ui/) | [Repo](https://github.com/dc-release/dc-portal/tree/v1beta) | EKS dev with internal ingress
```
Comment (ayasuda-ge) [inline README.md:48]: please add back and reconcile.

File: src/middleware/ManageApi.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+// import React from 'react';
+import axios from "axios";
+
+let Host = "https://dev-threadconnect.apps.ge.com/v1/api";
```
Comment (ayasuda-ge) [inline src/middleware/ManageApi.js:4]: this must go in the embedded db.

File: src/containers/threadConnect/Users.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
+import React, { useState } from "react";
+import { Row, Col, Table } from "react-bootstrap";
 
 import defaultLogo from "../../assets/images/Logo-UserProfile.jpg";
 
 export default function Users() {
-  const tableData = [
+  const initialTableData = [
     {
       userName: "Shubham Paskanti",
```
Comment (ayasuda-ge) [inline src/containers/threadConnect/Users.js:9]: this needs to go

File: src/containers/threadConnect/Users.js [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
       userId: "503249986",
```
Comment (ayasuda-ge) [inline src/containers/threadConnect/Users.js:10]: this needs to be persisted in the db

File: src/containers/threadConnect/APIs.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Row, Col, Table } from "react-bootstrap";
+
+import defaultLogo from "../../assets/images/Logo-UserProfile.jpg";
+
+export default function APIs() {
+  const tableData = [
```
Comment (ayasuda-ge) [inline src/containers/threadConnect/APIs.js:7]: persist in db

**Review by ayasuda-ge — CHANGES_REQUESTED** [review]
this file breaches the security practice https://github.com/dkrypt/dc-portal/blob/e7429412d58b8cadf959f61a4cfe4aa64164e7d6/src/containers/manage/threadConnect/UpdateManagement.js#L10 @dkrypt @paskantishubham

File: src/containers/threadConnect/IntegrationVersions.js [hunk 8]
```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { Row, Col, Table, Dropdown } from "react-bootstrap";
+
+import defaultLogo from "../../assets/images/Logo-UserProfile.jpg";
+
+export default function IntegrationVersions() {
+  const [integration, setIntegration] = useState("");
+  const [integrationArr, setIntegrationArr] = useState([]);
+
+  const tableData = {
```
Comment (ayasuda-ge) [inline src/containers/threadConnect/IntegrationVersions.js:10]: This goes in the db.

**Comment by ayasuda-ge** [comment]
> This PR introduces latest changes from developers contributing in GE GitHub Enterprise Repository.

What are the issues this "latest changes" addressed to? @dkrypt @paskantishubham

File: src/containers/threadConnect/Users.js (hunk 6 above)
Comment (paskantishubham) [reply src/containers/threadConnect/Users.js:10]: @ayasuda-ge This is used just for the mock up. Will move it to APIs once its ready

File: src/containers/threadConnect/APIs.js (hunk 7 above)
Comment (paskantishubham) [reply src/containers/threadConnect/APIs.js:7]: @ayasuda-ge This is used just for the mock up. Will move it to APIs once its ready

File: src/containers/threadConnect/IntegrationVersions.js (hunk 8 above)
Comment (paskantishubham) [reply src/containers/threadConnect/IntegrationVersions.js:10]: @ayasuda-ge This is used just for the mock up. Will move it to APIs once its ready

File: src/middleware/ManageApi.js (hunk 4 above)
Comment (ayasuda-ge) [reply src/middleware/ManageApi.js:4]: consider this-
```
data.insert("my-key",{"host":"https://dev-threadconnect.apps.ge.com/v1/api"});

var op = data.get("my-tc-call-info");

- document.Host=op.host;
let ProjectNameListUrl = _`${Host}/tc/project-names`;_


```

**Comment by dkrypt** [comment]
@ayasuda-ge  I would propose to close this PR, as it does not met the standards of git and also general engineering practices. 
Also, exposes some of the API Endpoints which are unprotected at the moment. This will lead to exposing these endpoints in the history.

I will propose a PR with new standard changes.

**Pull request closed without merging**